package.json

```diff
@@ -13,10 +13,10 @@
   "dependencies": {
     "react": "18.0.0",
     "react-dom": "18.0.0",
-    "react-scripts": "4.0.3"
+    "react-scripts": "^5.0.1"
   },
   "devDependencies": {
-    "@craco/craco": "6.4.5",
+    "@craco/craco": "^7.1.0",
     "@craco/types": "7.1.0",
     "@types/node": "16.11.7",
```

Comment:
@a-ignatov-parc  this I did not update - should I update it also considering update of node version? ATM it does not error. I've tried finding a connection between the node version and `@types/node` but could not find one (there was something about major and minor versions to correspond but for our version of node i could not find a minor version equivalent in `@types/node`)

Reply:
@r-bartolomei, yes, we need to update it as some APIs might change between versions, and we want our types to be aligned with these changes.

> but could not find one (there was something about major and minor versions to correspond but for our minor version of node i could not find a minor version equivalent in `@types/node`)

The general rule for Node.js is to find the latest version for major and minor versions (`18.12.X`). If it doesn't exist, we need to pick the latest versions for the previous minor (`18.11.X`). This means that there were no changes in the API and types.